package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maas-schemas",
-  "version": "9.3.0",
+  "version": "9.3.1",
```

Comment:
9.3.1 is current schema version, please rebase and bump to 9.3.2

Reply:
Fixed.